services/consensus_service.proto

```diff
@@ -31,93 +38,89 @@ import "transaction_response.proto";
- * be signed by the autoRenewAccount.
- *
- * The autoRenewPeriod on an account must currently be set a value in createTopic between
- * MIN_AUTORENEW_PERIOD (6999999 seconds) and MAX_AUTORENEW_PERIOD (8000001 seconds). During
- * creation this sets the initial expirationTime of the topic (see more below).
- *
- * If no adminKey is on a topic, there may not be an autoRenewAccount on the topic, deleteTopic is
- * not allowed, and the only change allowed via an updateTopic is to extend the expirationTime.
- *
- * If an adminKey is on a topic, every updateTopic and deleteTopic transaction must be signed by the
- * adminKey, except for updateTopics which only extend the topic's expirationTime (no adminKey
- * authorization required).
- *
- * If an updateTopic modifies the adminKey of a topic, the transaction signatures on the updateTopic
- * must fulfill both the pre-update and post-update adminKey signature requirements.
+ * The Hedera Consensus Service (HCS) provides the ability for a Hashgraph to
+ * provide aBFT consensus as to the order and validity of messages submitted to
+ * a *topic*, as well as a *consensus timestamp* for those messages.
  *
- * Mirrornet ConsensusService may be used to subscribe to changes on the topic, including changes to
- * the topic definition and the consensus ordering and timestamp of submitted messages.
- *
- * Until autoRenew functionality is supported by HAPI, the topic will not expire, the
- * autoRenewAccount will not be charged, and the topic will not automatically be deleted.
- *
- * Once autoRenew functionality is supported by HAPI:
- *
- * 1. Once the expirationTime is encountered, if an autoRenewAccount is configured on the topic, the
- * account will be charged automatically at the expirationTime, to extend the expirationTime of the
- * topic up to the topic's autoRenewPeriod (or as much extension as the account's balance will
- * supply).
- *
- * 2. If the topic expires and is not automatically renewed, the topic will enter the EXPIRED state.
- * All transactions on the topic will fail with TOPIC_EXPIRED, except an updateTopic() call that
- * modifies only the expirationTime.  getTopicInfo() will succeed. This state will be available for
- * a AUTORENEW_GRACE_PERIOD grace period (7 days).
- *
- * 3. After the grace period, if the topic's expirationTime is not extended, the topic will be
- * automatically deleted and no transactions or queries on the topic will succeed after that point.
  */
 service ConsensusService {
     /**
-     * Create a topic to be used for consensus.
-     * If an autoRenewAccount is specified, that account must also sign this transaction.
-     * If an adminKey is specified, the adminKey must sign the transaction.
-     * On success, the resulting TransactionReceipt contains the newly created TopicId.
-     * Request is [ConsensusCreateTopicTransactionBody](#proto.ConsensusCreateTopicTransactionBody)
+     * Create an HCS topic.
+     * <p>
```

Comment:
Maybe we could add a section on Service definitions to the Style Guidelines. Seems messages cannot contain `<p>` tags but Services can is that true?

Reply:
It's tricky because gRPC service syntax is somewhat "bolted on" and reuses other proto syntax with different keywords.
Basically, a `service` is equivalent to a `message`; a `rpc` is equivalent to a `field`.

I'll add something to that effect to the guidelines.

Reply:
Updated.